ignite/metrics/mean_average_precision.py

```diff
@@ -0,0 +1,89 @@
+__all__ = ["MeanAveragePrecision"]
+
+from typing import Callable, List, Optional, Sequence, Union
+
+import torch
+from torchvision.ops import box_iou
```

Comment:
Let's do something like that for torchvision dependency:
https://github.com/pytorch/ignite/blob/a0383ff22a98e53b556382a29a06bb9e2995645c/ignite/metrics/gan/fid.py#L169-L172